pages/common/duf.md

```diff
@@ -0,0 +1,20 @@
+# duf
+
+> Disk Usage/Free Utility - a better 'df' alternative.
+> More information: <https://github.com/muesli/duf>.
+
+- List accessible devices:
+
+`duf`
+
+- List everything (including pseudo, duplicate, inaccessible file systems):
+
+`duf --all`
+
+- Only list specific devices & mount points:
+
+`duf /home /some/file`
+
+- Sort the output:
+
+`duf --sort size`
```

Comment:
```suggestion
`duf --sort {{size}}`
```

Are there any other options we can use here except size?

Reply:
> Valid keys are: mountpoint, size, used, avail, usage, inodes, inodes_used, inodes_avail, inodes_usage, type, filesystem.

Reply:
Thanks. I've added it in the last commit.